examples/Part 02 - Intro to Private Training with Remote Execution.ipynb

```diff
@@ -130,7 +130,18 @@
    "cell_type": "code",
```

Comment:
<p>This sounds a little too marketing-ish. I'd suggest something like:</p><p>&gt; If your model doesn't fit into the Sequential paradigm, you can use Keras's functional API, or even subclass [<code>tf.keras.Model</code>](<a href="https://www.tensorflow.org/guide/keras/custom_layers_and_models#building_models" target="_blank">https://www.tensorflow.org/guide/keras/custom_layers_and_models#building_models</a>) to create custom models.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft-TensorFlow/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
sounds good!

Reply:
fixed